### PR TITLE
Remove commented out call to toolMappingFile

### PR DESCRIPTION
**Pull request by pfuehrlich-pik**

### CODE REVIEW

**Review by Renato-Rodrigues — COMMENTED** [review]
Is the file "mappingGAINStoREMINDsectors.csv" shipped with REMIND?
Wouldn't it be better to use the above line instead, and load the file from the input/mappings folder avoiding shipping this file together with the model?

**Comment by pfuehrlich-pik** [comment]
I have absolutely no idea :D I was just removing toolMappingFile everywhere because it's long deprecated, and Jan suggested to remove commented out code with toolMappingFile entirely.

**Comment by Renato-Rodrigues** [comment]
@giannou or @LaviniaBaumstark could one of you comment on this? 
I think it would be better practice to use the commented line instead to avoid adding an unnecessary file duplication and extra dependency to remind, no?

**Comment by giannou** [comment]
Since `madrat` is already needed to run REMIND we might as well use it, so I am also in favor of using the commented out line. The file mappingGAINStoREMINDsectors.csv is shipped with REMIND, yes.

**Comment by LaviniaBaumstark** [comment]
I also agree

File: scripts/input/exoGAINSAirpollutants.R [hunk 1]
```diff
@@ -138,7 +138,6 @@ E <- mbind(E,dimSums(E,dim=1))
 # E_calibrated <- E * E(2015) / E_CEDS(2015)
 
 # read mapping from GAINS sectors to REMIND sectors
-#map_GAINSsec2REMINDsec <- read.csv(madrat:::toolMappingFile("sectoral", "mappingGAINStoREMINDsectors.csv"), stringsAsFactors=FALSE,na.strings = "")
 map_GAINSsec2REMINDsec <- read.csv("mappingGAINStoREMINDsectors.csv", stringsAsFactors=FALSE,na.strings = "")
```
Comment (pfuehrlich-pik) [inline scripts/input/exoGAINSAirpollutants.R:141]:
```suggestion
map_GAINSsec2REMINDsec <- read.csv(madrat::toolMappingFile("sectoral", "mappingGAINStoREMINDsectors.csv"), stringsAsFactors=FALSE, na.strings = "")
```

Comment (pfuehrlich-pik) [reply scripts/input/exoGAINSAirpollutants.R:141]: Just commit and merge if that's what you mean :)

**Review by Renato-Rodrigues — APPROVED** [review]

**Comment by pfuehrlich-pik** [comment]
I don't have permission to merge in this repo, so can someone with permission please do it?

**Pull request merged**